src/pages/PeoplePage.tsx

```
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+  const { slug } = useParams();
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        const data = await getPeople();
+
+        setPeople(data);
+      } catch {
+        setError('Failed to fetch people');
+      } finally {
+        setIsLoading(false);
```

In the `catch` block you currently use `catch { ... }` which swallows the actual error. To aid debugging and provide a better error message consider changing to `catch (err) { console.error(err); setError(err?.message || 'Failed to fetch people'); }`. This preserves the existing UI message while exposing the original error for diagnostics. This change addresses useful error reporting but is not required for functionality.